lib/easypost/models/shipment.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class EasyPost::Models::Shipment < EasyPost::Models::EasyPostObject
+  # Get the lowest rate of a Shipment (can exclude by having `'!'` as the first element of your optional filter lists).
+  def lowest_rate(carriers = [], services = [])
+    EasyPost::Util.get_lowest_object_rate(self, carriers, services)
+  end
+
+  # Get the lowest smartrate of a Shipment.
+  def lowest_smartrate(delivery_days, delivery_accuracy)
+    smartrates = get_smartrates
+    EasyPost::Util.get_lowest_smartrate(smartrates, delivery_days, delivery_accuracy)
+  end
```

Comment:
issue: this function shouldn't exist on the shipment model? Getting the lowest smartrate requires passing in the ID of the shipment, see Python, PHP, or Node as examples

Reply:
Yes, you're right. This will get removed.